cs/mailing.texy

```diff
@@ -87,7 +87,7 @@ $mail->addAttachment('example.txt', 'Hello John!');
 Šablony
 -------
 
-Pokud posíláte HTML e-maily, přímo se nabízí je zapisovat v šablonovacím systému [Latte|latte:]. Jak na to?
+Pokud posíláte HTML e-maily bez maker, které jsou dostupné pouze v Nette Application, přímo se nabízí je zapisovat v šablonovacím systému [Latte|latte:]. Jak na to?
```

Comment:
👎 ta věta takhle nedává vůbec smysl. nemůže začít úvod kapitoli tím, že zmíníš nějaký random detail co teď nikoho nezajímá

Reply:
`jakým způsobem bys upozotrnil uživatele?`

Reply:
já taky neumím moc česky. zkusil bych to dát jako další větu nebo odstavec.
taky je lepší se soustředit na pozitiva ne na negativa.
kromě toho nejsem ani přesvědčen, že je uživatele na to potřeba vůbec explicitně upozornit